.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full
+
+# Install custom tools, runtimes, etc.
+# For example "bastet", a command-line tetris clone:
+# RUN brew install bastet
+#
+# More information: https://www.gitpod.io/docs/config-docker/
+RUN pyenv install 3.7.2
+RUN pyenv local 3.7.2
+RUN pip install black
```

Comment:
This seems to not be reflected in the test pod you shared with me. I get this error:

```
black --check samcli/* tests/*
pyenv: black: command not found

The `black' command exists in these Python versions:
  3.7.2

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
make: *** [Makefile:46: black-check] Error 127
```

Reply:
indeed as per https://www.gitpod.io/docs/languages/python/ `pyenv global` should be used rather than `pyenv local` to have the resulting shell presented with the desired version (which includes black)

Reply:
updated PR and tested, 

```
============================================================== 1525 passed, 4 warnings in 173.43s (0:02:53) ================================================================
black --check samcli/* tests/*
All done! ✨ 🍰 ✨
371 files would be left unchanged.
gitpod /workspace/aws-sam-cli $ python --version
Python 3.7.2
gitpod /workspace/aws-sam-cli $ black --check samcli/* tests/*

All done! ✨ 🍰 ✨
371 files would be left unchanged.
gitpod /workspace/aws-sam-cli $ 
gitpod /workspace/aws-sam-cli $ 
```